app/src/main/java/com/mghelas/internship_playground/sweetsfactory/Main.java

```diff
@@ -0,0 +1,25 @@
+package com.mghelas.internship_playground.sweetsfactory;
+
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.List;
+
+public class Main {
+    public static void main(String[] args) {
```

Comment:
do you still need Main class since you create your chocolates in activities ?

Reply:
no. But i can leave it for test purposes

Reply:
ok